specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -0,0 +1,2618 @@
+        "tags": [
+          "amlFilesystems"
+        ],
+        "description": "Schedules a amlFilesystem for deletion.",
+        "operationId": "amlFilesystems_Delete"
+      },
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "description": "Target resource group.",
+            "in": "path",
+            "name": "resourceGroupName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref":"#/parameters/amlFilesystemNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the amlFilesystem object corresponding to amlFilesystemName.",
+            "schema": {
+              "$ref": "#/definitions/amlFilesystem"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "amlFilesystems_Get": {
+            "$ref": "./examples/amlFilesystems_Get.json"
+          }
+        },
+        "tags": [
+          "amlFilesystems"
+        ],
+        "description": "Returns an amlFilesystem.",
+        "operationId": "amlFilesystems_Get"
+      },
+      "put": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "description": "Target resource group.",
+            "in": "path",
+            "name": "resourceGroupName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref":"#/parameters/amlFilesystemNameParameter"
+          },
+          {
+            "description": "Object containing the user-selectable properties of the new amlFilesystem. If read-only properties are included, they must match the existing values of those properties.",
+            "in": "body",
+            "name": "amlFilesystem",
+            "schema": {
+              "$ref": "#/definitions/amlFilesystem"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "amlFilesystem created or updated.",
+            "schema": {
+              "$ref": "#/definitions/amlFilesystem"
+            }
+          },
+          "201": {
+            "description": "amlFilesystem creation or update has been initiated. Poll the new amlFilesystem's provisioningState property to monitor creation progress.",
```

Comment:
When we are doing namespace hydration, the provisioning state may got to succeeded a long time before the filesystem is ready for use. Maybe we should say to poll the provisioning state and health.

Reply:
Fixed.